HACKING.rst.txt

```diff
@@ -8,6 +8,17 @@ community process works in practice.  Readers who are familiar with the SciPy
 community and are experienced Python coders may want to jump straight to the
 `git workflow`_ documentation.
 
+There are a lot of ways you can contribute:
+
+- Contributing new code/features
+- Fixing bugs and other maintenance work
+- Improving the documentation
+- Reviewing open pull requests
```

Comment:
perhaps this should `:ref:` to the title below, same with the "Contributing new code"